client/src/components/NotFound.js

```diff
@@ -11,7 +11,7 @@ export default class NotFound extends Component {
 
 	render() {
 		return <div>
-            <h1 className="not-found-text">{this.props.text}</h1>
+            <h1 style={{ textAlign: 'center' }}>{this.props.text}</h1>
```

Comment:
Why is this changed from a class to an in-line style?

Reply:
For styling it is preferred to use inline styles. This reduces dependencies to implicit style definitions and more..[https://github.com/necolas/react-native-web/blob/master/docs/guides/style.md](.https://github.com/necolas/react-native-web/blob/master/docs/guides/style.md)

Reply:
Then I would prefer to see it in its own separate commit with a clear message, since it bears no direct relation to the referenced issue.